proposals/lip-0041.md

```diff
@@ -268,25 +469,23 @@ Notice that a transaction in the transaction pool with a nonce greater than the
 
 #### Before Command Execution
 
-Before the command referenced in a transaction `trs` is executed, the following protocol logic is applied.
-
-* Create an auth account if no account exists for the sender address:
+Before the command referenced in a transaction `trs` is executed, an auth account is created if no account exists for the sender address.
 
 ```python
-let senderAddress be the address corresponding to trs.senderPublicKey
-if there is no entry in the in the auth data substore with storeKey == senderAddress:
-  create an entry in the auth data substore with storeKey = senderAddress, initialized to default values
+def beforeCommandExecution(trs: Transaction) -> None:
```

Comment:
Is there is a specific reason why this logic is performed here and split from incrementing the nonce?

Reply:
not so sure anymore, maybe to have an easier nonce check in the transaction verification? although that is called also in the transaction pool, so we can not really skip the check on the account existence anyway

Reply:
@shuse2 Do you now the reason for it?

Not sure if it makes a performance difference if we modify the same state entry twice instead of once (I assume it would be cached).

Reply:
performance-wise, there is very little difference because it will be cached.
I think we could just increment here as well if there is no specific reason

Reply:
done